docs/source/dev_setup.md

```diff
@@ -136,13 +136,11 @@ For more information refer to the official [Reference on Cloud Function for Fire
 
 ## Travis Setup
 
-Configuration for travis setup is utilizing environment variables.
+A Travis instance is used to build MapSwipe Workers and run tests.
 
-One difference to production or development setup is that the service account key as json is stored in the environment variables `FIREBASE_CONFIG` as text. To make this work special characters as to be escaped first. This command will simply escape every character:
+For the configuration of Travis environment variables are used: https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings
```

Comment:
could you add here which environment variables are set in travis. I think we only set some and not all. E.g. `bing_imagery_api_key` or so is not set and doesn't need to be set.

Reply:
Sure. I added more information on the travis setup to the docs.